File: examples-sdk/kotlin/026-basic-ai-agent/main.kt
// 026-basic-ai-agent — Kotlin SDK equivalent
// Compile: vil compile --from kotlin --input 026-basic-ai-agent/main.kt --release

fun main() {
    val server = VilServer("ai-agent", 8080)
    val agent = ServiceProcess("agent")
    agent.endpoint("POST", "/agent", "agent_handler")
    server.service(agent)
    server.compile()
}

File: examples-sdk/swift/026-basic-ai-agent/main.swift
// 026-basic-ai-agent — Swift SDK equivalent
// Compile: vil compile --from swift --input 026-basic-ai-agent/main.swift --release

let server = VilServer(name: "ai-agent", port: 8080)
let agent = ServiceProcess(name: "agent")
agent.endpoint(method: "POST", path: "/agent", handler: "agent_handler")
server.service(agent)
server.compile()
